### PR TITLE
some minor fixes

### PR DESCRIPTION
**Pull request by DiegoAndresCortes**

### CODE REVIEW

**Comment by live627** [comment]
I initiated a rebase a few days ago. Lots of merge conflicts needing a manual review; therefore, you'll need to rebase  this once I finally push the updated commits.

**Comment by DiegoAndresCortes** [comment]
Alright, thanks

**Comment by DiegoAndresCortes** [comment]
Didn't look right before so I copied the branch and cherry picked back the commits

File: Themes/default/css/index.css [hunk 1]
```diff
@@ -2914,7 +2895,7 @@ tr.windowbg td, tr.bg td, .table_grid tr td {
 /* Styles for the profile section.
 ------------------------------------------------- */
 /* The basic user info on the left */
-#profileview {
+#profileview.summary {
```
Comment (DiegoAndresCortes) [inline Themes/default/css/index.css]: What do you think of this, I'm not convinced it should have a different selector? Perhaps I should be using :has() to check for basicinfo and detailedinfo?

Comment (live627) [reply Themes/default/css/index.css]: Perhaps the stats panel could drop that identifier

File: Themes/default/Errors.template.php [hunk 2]
```diff
@@ -42,7 +42,7 @@ function template_fatal_error()
 				', Utils::$context['error_title'], '
 			</h3>
 		</div>
-		<div class="windowbg" ', Utils::$context['error_code'], '>
+		<div class="windowbg ', Utils::$context['error_code'], '">
```
Comment (live627) [inline Themes/default/Errors.template.php]:
```suggestion
		<div class="windowbg" id="http_', Utils::$context['error_code'], '">
```

**Pull request merged**